libsel4vm/src/arch/x86/hpet.c

```diff
@@ -0,0 +1,737 @@
+/*
```

Comment:
Would it be better to put this file in https://github.com/seL4/camkes-vm/tree/master/components/Init/src along with other device emulators?

Reply:
That is where we had it originally, but it got moved to the library. I'm not sure why, but I can try to move it back.

Reply:
I moved it back to the Init component.